python/GafferImageUI/CatalogueUI.py

```diff
@@ -281,9 +282,11 @@ def _updateFromPlug( self ) :
 		images = self.__images()
 		if len( images ) :
 			image = images[index % len( images )]
-			self.__pathListing.setSelection( IECore.PathMatcher( [ "/" + image.getName() ] ) )
-			self.__descriptionWidget.setPlug( image["description"] )
-			self.__nameWidget.setGraphComponent( image )
+			indices = self.__indicesFromSelection()
+			if index not in indices :
+				self.__pathListing.setSelection( IECore.PathMatcher( [ "/" + image.getName() ] ) )
```

Comment:
When selecting an image, the "Name" and "Description" fields are no longer updating to reflect the selection. I suspect this is because they need to be outside the conditional here.